pkg/cli/options.go

```diff
@@ -171,26 +171,44 @@ func parseExternalPluginArgs() (args []string) {
 	return args
 }
 
+// isHostSupported checks whether the host system is supported or not.
+func isHostSupported(host string) bool {
+	for _, platform := range supportedPlatforms {
+		if host == platform {
+			return true
+		}
+	}
+	return false
+}
+
 // getPluginsRoot detects the host system and gets the plugins root based on the host.
 func getPluginsRoot(host string) (pluginsRoot string, err error) {
```

Comment:
/hold
```suggestion
func getPluginsRoot(host string) {
```

Can we first remove the return since it is not used? 
https://github.com/kubernetes-sigs/kubebuilder/pull/3247/files#r1126146318

Reply:
@camilamacedo86 this function is assigned to [`retrievePluginsRoot`](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/cli/options.go#L39), and then called by [`DiscoverExternalPlugins`](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/cli/options.go#L198). the output of the function is indeed used then to lookup external plugins in `pluginsRoot`.
to elaborate as to why I've changed `return pluginsRoot, nil` to just `return`: it's because the function uses named returns, so while using `return pluginsRoot, nil` as a return expression would work, specifying the variables would be redundant, so essentially, just using a "void" `return` will have the same outcome (because of the named return in the signature).
but either way, I can revert it back to the old style as it would work fine as well. what do you think?
thank you!

Reply:
I think that is fine as it is . We can re-check in a follow up either.
Thank you a lot for your help to address those ones.

Reply:
thank you @camilamacedo86, always happy to help!